README.md

```diff
@@ -66,119 +65,95 @@ or
-      source 'https://github.com/CocoaPods/Specs.git'
+      # Point to the installed version
+      pod 'RNDateTimePicker', :path => '../node_modules/@react-native-community/datetimepicker/RNDateTimePicker.podspec'
 
-      target 'MyApp' do
-        # As we use Swift, ensure that `use_frameworks` is enabled.
-        use_frameworks!
+      # React/React-Native specific pods
+      pod 'React', :path => '../node_modules/react-native', :subspecs => [
+        'Core',
+        'CxxBridge',      # Include this for RN >= 0.47
+        'DevSupport',     # Include this to enable In-App Devmenu if RN >= 0.43
+        'RCTText',
+        'RCTNetwork',
+        'RCTWebSocket',   # Needed for debugging
+      ]
 
-        # Specific iOS platform we are targetting
-        platform :ios, '8.0'
+      # Explicitly include Yoga if you are using RN >= 0.42.0
+      pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
 
-        # Point to the installed version
-        pod 'RNDateTimePicker', :path => '../node_modules/@react-native-community/datetimepicker/RNDateTimePicker.podspec'
+      # Third party deps podspec link
+      pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+      pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
+      pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
-        # React/React-Native specific pods
-        pod 'React', :path => '../node_modules/react-native', :subspecs => [
-          'Core',
-          'CxxBridge',      # Include this for RN >= 0.47
-          'DevSupport',     # Include this to enable In-App Devmenu if RN >= 0.43
-          'RCTText',
-          'RCTNetwork',
-          'RCTWebSocket',   # Needed for debugging
-        ]
-
-        # Explicitly include Yoga if you are using RN >= 0.42.0
-        pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
-
-        # Third party deps podspec link
-        pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
-        pod 'glog', :podspec => '../node_modules/react-native/third-party-podspecs/glog.podspec'
-        pod 'Folly', :podspec => '../node_modules/react-native/third-party-podspecs/Folly.podspec'
-
-      end
-      ```
-  4. Run `pod install` inside the same folder where the `pod` file was created
-  5. `npm run start`
-  6. `npm run start:ios`
+    end
+    ```
+4. Run `pod install` inside the same folder where the `pod` file was created
+5. `npm run start`
+6. `npm run start:ios`
 
 #### Android
 
 1. Add the following lines to `android/settings.gradle`:
-
    ```gradle
    include ':react-native-datetimepicker'
    project(':react-native-datetimepicker').projectDir = new File(rootProject.projectDir, '../node_modules/@react-native-community/datetimepicker/android')
    ```
 
-2. Add into `android/gradle.properties`:
```

Comment:
This is not needed anymore? `useAndroidX` and `enableJetifier`?

Reply:
This is now added by default by a React app, see https://react-native-community.github.io/upgrade-helper/?from=0.59.3&to=0.60.4 and scroll down to the file

Reply:
More background on the React@0.60 upgrade: https://facebook.github.io/react-native/blog/2019/07/03/version-60#androidx-support